index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- link for CSS -->
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <!-- tab title -->
+    <title>Horiseon | Marketing Agency</title>
+  </head>
+
+  <body>
+    <!-- header -->
+    <header>
+      <!-- company logo -->
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <nav>
+        <ul>
+          <!-- link 1 -->
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <!-- link 2 -->
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <!-- link 3 -->
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- hero image -->
```

Comment:
Consider added a blank line here to break up your HTML as it makes it easier to identify the distinct sections at a glance. It's not too much of an issue here as there isn't much code, but it's a good habit to have for when your code gets longer.